content/office-hours/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Office hours
+
+menu:
+  main:
+    parent: 'Community'
+    name: 'Office hours'
+---
+
+# PrestaShop Project's Office Hours
+
+The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.
+
+These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```

Comment:
I think it'd be best to keep the tone to a more "matter-of-fact", less opinionated style: what kinds of topics are discussed, why would it be interesting for me to participate?

Reply:
@eternoendless let me know if the new version is better